### PR TITLE
IOP kernel DeleteEventFlag test

### PR DESCRIPTION
**Pull request by jpd002**
Basic test for DeleteEventFlag kernel function.

### CODE REVIEW

**Comment by jpd002** [comment]
Also, since you've given me "Collaborator" rights, how should we proceed if I want to submit changes to this repo? Do you still want me to use pull requests?

File: tests/kernel/iop/eventflag/delete.expected [hunk 1]
```diff
@@ -0,0 +1,10 @@
+-- TEST BEGIN
+unused event flag:
+  delete after creation: succeeded
+used event flag:
+  delete while thread is waiting: succeeded
+  WaitEventFlag returned: -425
+  DeleteThread returned: 0
```
Comment (unknownbrackets) [inline tests/kernel/iop/eventflag/delete.expected:7]: For illustration, what about deleting twice?

-[Unknown]

**Review by unknownbrackets — CHANGES_REQUESTED** [review]
Well, I think it's still best to review, but figured you should have access.

-[Unknown]

File: tests/kernel/iop/eventflag/delete.expected (hunk 1 above)
Comment (jpd002) [reply tests/kernel/iop/eventflag/delete.expected:7]: I've pushed a commit that adds that test case.

Comment (unknownbrackets) [reply tests/kernel/iop/eventflag/delete.expected:7]: Thanks.

-[Unknown]

**Comment by jpd002** [comment]
Thanks for merging!

**Pull request merged**